app/TracingStrategyContext.tsx

```diff
@@ -4,10 +4,16 @@ import { ExposureHistoryProvider } from './ExposureHistoryContext';
 
 interface TracingStrategyContextState {
   name: string;
+  homeScreenComponent: () => JSX.Element;
 }
 
+const DefaultHomeScreenComponent = () => {
+  return <></>;
+};
+
 const defaultState = {
   name: 'initializingTracingStrategy',
+  homeScreenComponent: DefaultHomeScreenComponent,
```

Comment:
Not a fan of having the homescreen component on the tracing strategy. Ideally a future commit could separate the data required by the tracing strategy and the application view logic.